src/app/lib/utilities/preprocessor/rules/convertHeadingsToSubheadings.js

```diff
@@ -0,0 +1,23 @@
+import R from 'ramda';
```

Comment:
```suggestion
import { clone } from 'ramda';
```
I think this helps tree shake too :)